### PR TITLE
Improve grammar and clarity of engine descriptions

### PR DESCRIPTION
**Pull request by NovemberOrWhatever**
I do not know the correct format of pull requests, but I've edited part descriptions to improve grammar.

### CODE REVIEW

File: GameData/RealismOverhaul/Engine_Configs/Castor_30XL_Config.cfg [hunk 1]
```diff
@@ -217,4 +217,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Castor-30XL] { %ratedBurnTime = #$/TESTFLIGHT[Castor-30XL]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Castor_30XL_Config.cfg:220]: Needs a newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/RD0242M2_Config.cfg [hunk 2]
```diff
@@ -120,4 +120,4 @@
 	}
 
 	@MODULE[ModuleEngineConfigs] { @CONFIG[RD-0242M2] { %ratedBurnTime = #$/TESTFLIGHT[RD-0242M2]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/RD0242M2_Config.cfg:123]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/RD111_config.cfg [hunk 3]
```diff
@@ -136,4 +136,4 @@
 		cycleReliabilityEnd = 0.952604
 		techTransfer = RD-108-8D75-1958,RD-108-8D77,RD-108-8D75PS,RD-108-8D75:20
 	}
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/RD111_config.cfg:139]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/RD1_Config.cfg [hunk 4]
```diff
@@ -140,4 +140,4 @@
 	}
 
 	@MODULE[ModuleEngineConfigs] { @CONFIG[RD-1] { %ratedBurnTime = #$/TESTFLIGHT[RD-1]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/RD1_Config.cfg:143]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/RD57_Config.cfg [hunk 5]
```diff
@@ -185,4 +185,4 @@
 		ignitionDynPresFailMultiplier = 0.1
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[RD-57M] { %ratedBurnTime = #$/TESTFLIGHT[RD-57M]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/RD57_Config.cfg:188]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/STBE1_Config.cfg [hunk 6]
```diff
@@ -254,4 +254,4 @@
 		cycleReliabilityEnd = 0.99995
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[STBE-3] { %ratedBurnTime = #$/TESTFLIGHT[STBE-3]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/STBE1_Config.cfg:257]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star13B_Config.cfg [hunk 7]
```diff
@@ -324,4 +324,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-13B] { %ratedBurnTime = #$/TESTFLIGHT[Star-13B]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star13B_Config.cfg:327]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star20_Config.cfg [hunk 8]
```diff
@@ -327,4 +327,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-20] { %ratedBurnTime = #$/TESTFLIGHT[Star-20]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star20_Config.cfg:330]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star27_Config.cfg [hunk 9]
```diff
@@ -571,4 +571,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-27] { %ratedBurnTime = #$/TESTFLIGHT[Star-27]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star27_Config.cfg:574]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star3_Config.cfg [hunk 10]
```diff
@@ -220,4 +220,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-3] { %ratedBurnTime = #$/TESTFLIGHT[Star-3]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star3_Config.cfg:223]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star5C_Config.cfg [hunk 11]
```diff
@@ -322,4 +322,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-5C] { %ratedBurnTime = #$/TESTFLIGHT[Star-5C]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star5C_Config.cfg:325]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star5D_Config.cfg [hunk 12]
```diff
@@ -116,4 +116,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-5D] { %ratedBurnTime = #$/TESTFLIGHT[Star-5D]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star5D_Config.cfg:119]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star6B_Config.cfg [hunk 13]
```diff
@@ -324,4 +324,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-6B] { %ratedBurnTime = #$/TESTFLIGHT[Star-6B]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star6B_Config.cfg:327]: Needs newline before EOF

File: GameData/RealismOverhaul/Engine_Configs/Star8_Config.cfg [hunk 14]
```diff
@@ -116,4 +116,4 @@
 		isSolid = True
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[Star-8] { %ratedBurnTime = #$/TESTFLIGHT[Star-8]/ratedBurnTime$ } }
-}
\ No newline at end of file
+}
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/Star8_Config.cfg:119]: Needs newline before EOF

**Comment by Capkirk123** [comment]
Other than a few missing newlines, looks good!

File: GameData/RealismOverhaul/Engine_Configs/RD1_Config.cfg [hunk 15]
```diff
@@ -31,7 +31,7 @@
 {
 	%title = RD-1
 	%manufacturer = Glushko
-	%description = Four-chamber nitric acid and kerosene developed by Glushko duting WW2. Used in various rocket powered aircraft proposals, and tested extensivley in the Pe-2RD, a Pe-2 bomber with an RD-1 installed in the fuselage. Abandoned after WW2, as jet engines proved to be more reliable for high performance aircraft.
+	%description = A four-chamber nitric acid and kerosene engine developed by Glushko duting WW2. It was used in various rocket powered aircraft proposals, and tested extensivley in the Pe-2RD, a Pe-2 bomber with an RD-1 installed in the fuselage, but abandoned after WW2, as jet engines proved to be more reliable for high performance aircraft.
```
Comment (DRVeyl) [inline GameData/RealismOverhaul/Engine_Configs/RD1_Config.cfg:35]: Dute!  [During...]

File: GameData/RealismOverhaul/Engine_Configs/KVD1_Config.cfg [hunk 16]
```diff
@@ -66,7 +66,7 @@
...
-	%description = Staged combustion hydrolox upper stage engine intended for use on the N-1M and considered for use on Proton and Angara, eventually licensed to India for its GSLV MkI and MkII.	Later versions were developed by India for domestic use on the as the CE-7.5.  The main engine bell is fixed in place, and two verniers are used to provide combined pitch-yaw-roll control; this leads to lower control authority than on stages where the main engine can gimbal.	 This engine runs with a somewhat higher than average O/F ratio, resulting in a denser than average hydrolox stage.
+	%description = A staged combustion hydrolox upper stage engine intended for use on the N-1M and considered for use on Proton and Angara, eventually licensed to India for its GSLV MkI and MkII.	Later versions were developed by India for domestic use on the as the CE-7.5.  The main engine bell is fixed in place and two verniers are used to provide combined pitch-yaw-roll control; this leads to lower control authority than on stages where the main engine can gimbal.	 This engine runs with a somewhat higher than average O/F ratio, resulting in a denser than average hydrolox stage.
```
Comment (al2me6) [inline GameData/RealismOverhaul/Engine_Configs/KVD1_Config.cfg:69]: Are the spaces between sentences intended?

File: GameData/RealismOverhaul/Engine_Configs/STBE_Config.cfg [hunk 17]
```diff
@@ -34,7 +34,7 @@
 {
 	%title = Space Transportation Booster Engine (STBE)
 	%manufacturer = Rocketdyne
-	%description = Late 1980s to Early 1990s Booster Engine Design Using JP-4 & Liquid Oxygen as its Propellant.
+	%description = A late 1980s to early 1990s booster engine design using JP-4 & Liquid Oxygen as its propellants.
```
Comment (al2me6) [inline GameData/RealismOverhaul/Engine_Configs/STBE_Config.cfg:37]: & -> and
Liquid Oxygen -> liquid oxygen

File: GameData/RealismOverhaul/Engine_Configs/SuperDraco_Config.cfg [hunk 18]
```diff
@@ -40,7 +40,7 @@
 	%category = Engine
 	%title = SuperDraco
 	%manufacturer = SpaceX
-	%description = A powerful hypergolic engine. Used by the Dragon V2 Command Module for powered landings, trajectory corrections and as a Launch Abort System (LAS). Other applications include hypersonic deceleration and landing for crew and cargo modules.
+	%description = A powerful hypergolic engine used by the Dragon V2 Command Module for powered landings, trajectory corrections and as a Launch Abort System (LAS). Other applications include hypersonic deceleration and landing for crew and cargo modules.
```
Comment (al2me6) [inline GameData/RealismOverhaul/Engine_Configs/SuperDraco_Config.cfg:43]: Factually incorrect/outdated. Change to:

> A powerful hypergolic engine used in the launch abort system of the Dragon 2 capsule. Other potential applications include hypersonic deceleration and landing for crew and cargo modules.

**Review by al2me6 — CHANGES_REQUESTED** [review]
Looks great overall! Tone is still sometimes inconsistent (we really need a style guide here), but certainly a significant improvement.

File: GameData/RealismOverhaul/Engine_Configs/KVD1_Config.cfg (hunk 16 above)
Comment (NovemberOrWhatever) [reply GameData/RealismOverhaul/Engine_Configs/KVD1_Config.cfg]: I believe it was how the original author did newlines. But I think it's better without

**Review by DRVeyl — APPROVED** [review]

**Pull request merged**